app/tests/test_amity.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+from app.cp.amity import Amity
+
+
+class Test_amity_class(unittest.TestCase):
+
+    def setUp(self):
+        self.amity = Amity()
+
+    def tearDown(self):
+        del self.amity
+
+    """
+    Tests for the create_room function in class Amity.
+    The tests validate input and output data and also see if
+    specified constraints are met and EdgeCases are handled. create_room
+    function takes the room_name and Room_Type as arguments.
+    """
+
+    def test_creating_room_with_wrong_room_type(self):
+        self.assertEqual(self.amity.create_room("Narnia", "Quiet_Room"),
+                         "Room_Type can only be OFFICE or LIVING_SPACE",
+                         msg="Room_Type can only be OFFICE or LIVING_SPACE")
+
+    def test_created_rooms_persisted(self):
```

Comment:
This seems like a redundant test. It is so specific to the code i.e relying on the structure of the code. If any change is made in the manner the rooms are stored, this test would fail, making it so reliant on the code. Consider dropping this test